src/plugin/translations/uploadTranslationsCommand.ts

```diff
@@ -240,40 +222,28 @@ export const uploadTranslations = (configHolder: CrowdinConfigHolder) => {
                     return;
                 }
 
+                const editor = vscode.window.activeTextEditor;
+                let branch = undefined
+                if(editor) {
+                    branch = CommonUtil.getCurrentGitBranch(editor.document.uri)
+                }
+                if (branch !== undefined) {
+                    branch = branch.replace(/[^\w\s-]/gi, "-")
+                }
+
```

Comment:
We need to protect the beta branch here. If the current branch is beta, reset it to undefined so that Crowdin default branch will be used

Reply:
Protected both beta and main